test/integration/master/kube_apiserver_test.go

```diff
@@ -190,6 +197,153 @@ func TestOpenAPIDelegationChainPlumbing(t *testing.T) {
 	}
 }
 
+func TestOpenAPIDelegationChainCRDProtection(t *testing.T) {
```

Comment:
nit: it's not really delegation chain protection, but only `ApiextensionsOverlapProtection`